core/src/test/java/io/cucumber/core/stepexpression/StepExpressionFactoryTest.java

```diff
@@ -158,12 +158,11 @@ void unknown_target_type_transform_doc_string_to_doc_string() {
     }
 
     @Test
-    void docstring_expression_transform_doc_string_with_content_type_to_string() {
+    void docstring_expression_transform_doc_string_to_string() {
```

Comment:
This test is now identical to the test above `unknown_target_type_transform_doc_string_to_doc_string`. Why did you change this?

Reply:
I can start by saying we can do without it, but they're not quite identical, they're covering both situations were you either want to convert the DocString that doesn't have a contentType to either a String (the default converter) or to an Object which gets converted to a DocString in io/cucumber/core/stepexpression/StepExpressionFactory.java:72

Reply:
Right. Makes sense.